iree/base/target_platform.h

```diff
@@ -104,12 +104,18 @@
 // IREE_PTR_SIZE_*
 //==============================================================================
 
-#if UINTPTR_MAX > UINT_MAX
-#define IREE_PTR_SIZE_64
-#define IREE_PTR_SIZE 8
+// See https://stackoverflow.com/q/51616057
+_Static_assert(sizeof(void*) == sizeof(uintptr_t),
```

Comment:
you can just use `static_assert` here instead of the funky casing

Reply:
Oh whew. I was wondering what the hell that was :-D This is what I get for copy-pasting from stack overflow

Reply:
Apparently I can't:

> ../iree/base/target_platform.h:108:15: error: expected parameter declarator
static_assert(sizeof(void*) == sizeof(uintptr_t),

https://source.cloud.google.com/results/invocations/e710f9e5-183d-4a0b-87d1-d945fd84f8bb/log